ansible/roles/avocado/tasks/main.yaml

```diff
@@ -0,0 +1,79 @@
+- name: Setup avocado repository
+  copy:
+    src: avocado.repo
+    dest: /etc/yum.repos.d/avocado.repo
+    owner: root
+    group: root
+    mode: 0644
+  tags:
+    - setup
+
+- name: Install packages
+  package: name={{item}}
+  notify: vt-bootstrap
+  with_items:
+    - avocado
+    - avocado-plugins-vt
+    - libvirt
+    - qemu-kvm
+    - wget
+  tags:
+    - setup
```

Comment:
As a side effect of this, I had to add a new role avocado-repo to configure avocado.repo before pkg_install role is called in avocado role. Otherwise, when pkg_install runs, it does not find avocado packages because avocado.repo is not set yet.